### DISCUSSION TITLE
Update for combat

### DISCUSSION THREAD

**Discussion by dan28000** [Zrips/CMI#7113] [Enhancement]
Add to combat whitelist for worlds where it would be enabled with commands for attacker and victim on start and end of fight

**Comment by mrfloris** [comment]
Sorry, I am not exactly sure what you're trying to say.

But, eventcommands.yml has pvp events you could perhaps use. Using specialized commands in the commands you can check if they're in a specific world as well.

**Comment by dan28000** [comment]
In combat section there would be whitelist/blacklist for worlds where the combat mode could be activated also with commands to run for each player at the end and start of combat

i know about event commands but there's no order what player started the combat

world1: you could activate combat
world2: can't activate combat at all
